config/indexer.go

```diff
@@ -17,6 +24,7 @@ func NewIndexer() Indexer {
 		CacheSize:      300000,
 		ValueStoreDir:  "valuestore",
 		ValueStoreType: "sth",
+		GCInterval:     Duration(30 * time.Minute),
```

Comment:
Would be good to make this configurable in future work.

Reply:
That's is just the default value.  It is configurable as a value in the config file.  Am I missing something?

Reply:
Nope it's me overlooking that; thanks